README.md

```diff
@@ -92,6 +119,11 @@ Define a function called `doSomething`. It should take one argument, called
 Your Answer:
 ```js
 // write code here
+function doSomething(thingToDo) {
+    alert("I have to " + thingToDo);
+}
+
+doSomething("go to the store.");
 ```
```

Comment:
`thingToDo` is an argument that will contain a function definition -- not a string. Here's an example of what we're looking for...

```js
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```